src/main/java/com/mraof/minestuck/entity/item/GristEntity.java

```diff
@@ -96,11 +118,73 @@ public boolean hurt(DamageSource source, float amount)
 		}
 	}
 	
+	public void setAnimationFromPacket(Animation animation)
+	{
+		switch(animation)
+		{
+			case REJECT:
+				shaderAlpha = 255;
+				break;
+		}
+	}
+	
+	public enum Animation
+	{
+		REJECT
+	}
+	
+	public class PlayerCanPickUpGristSelector implements Predicate<Entity>
+	{
+		private final GristEntity grist;
+		
+		public PlayerCanPickUpGristSelector(GristEntity gristEntity)
+		{
+			this.grist = gristEntity;
+		}
```

Comment:
Since you're writing the class as non-static, you don't need a field for the grist entity. You can instead access it directly or through `GristEntity.this`, replacing `this.grist`.